examples/medplum-demo-bots/src/billing-bots/cms-1500-test-data.ts

```diff
@@ -0,0 +1,352 @@
+      request: { method: 'POST', url: 'Claim' },
+      resource: {
+        resourceType: 'Claim',
+        identifier: [
+          {
+            system: 'http://example.org/claims',
+            value: 'example-claim',
+          },
+        ],
+        status: 'active',
+        type: {
+          coding: [
+            {
+              system: 'http://terminology.hl7.org/CodeSystem/claim-type',
+              code: 'professional',
+              display: 'Professional',
+            },
+          ],
+        },
+        use: 'claim',
+        patient: { reference: 'urn:uuid:8796eac7-4ee2-4b9a-84f7-68691173e945', display: 'Homer Simpson' },
+        created: '2020-04-17T11:15:12.614Z',
+        priority: {
+          coding: [
+            {
+              system: 'http://hl7.org/fhir/ValueSet/process-priority',
+              code: 'normal',
+            },
+          ],
+        },
+        insurance: [
+          {
+            coverage: {
+              reference: 'urn:uuid:72ec6c80-6dab-41e0-adff-a8670a5b4363',
+              display: 'Homer Simpson Health Insurance Plan',
+            },
+            sequence: 1,
+            focal: true,
+            preAuthRef: ['0923092390'],
+          },
+        ],
+        provider: { reference: 'urn:uuid:706245c5-5f9d-45eb-bf90-cee2fac3f52c' },
+        supportingInfo: [
+          {
+            category: {
+              coding: [
+                {
+                  code: 'employmentimpacted',
+                  system: 'http://terminology.hl7.org/CodeSystem/claiminformationcategory',
+                },
+              ],
+            },
+            sequence: 1,
+            timingPeriod: {
+              start: '2024-04-02',
+              end: '2024-04-20',
+            },
+          },
+          {
+            category: {
+              coding: [{ code: 'info', system: 'http://terminology.hl7.org/CodeSystem/claiminformationcategory' }],
+            },
+            sequence: 2,
+            code: {
+              coding: [{ system: 'http://example.org/info-codes', code: 'patientaccount' }],
+            },
+            valueString: '429802409',
+          },
+          {
+            category: {
+              coding: [{ system: 'http://terminology.hl7.org/CodeSystem/claiminformationcategory', code: 'onset' }],
+            },
+            sequence: 3,
+            timingDate: '2024-02-02',
+          },
+          {
+            category: {
+              coding: [
+                { system: 'http://terminology.hl7.org/CodeSystem/claiminformationcategory', code: 'hospitalized' },
+              ],
+            },
+            sequence: 4,
+            timingPeriod: {
+              start: '2024-03-30',
+              end: '2024-05-21',
+            },
+          },
+          {
+            category: {
+              coding: [
+                { system: 'http://terminology.hl7.org/CodeSystem/claiminformationcategory', code: 'patientpaid' },
```

Comment:
This is the system that is used as an example in FHIR, but the `patientpaid` code isn't actually in it